.github/aws-nuke.yaml

```diff
@@ -0,0 +1,149 @@
+regions:
+#- us-east-1
+- us-east-2
+
+account-blacklist:
+- "999999999999" # production
+
+resource-types:
+  # only nuke these three resources
+  targets:
+  - IAMRole
+  - S3Object
+  - S3Bucket
+  - EC2Address 
+  - ElasticBeanstalkApplication
+  - ElasticBeanstalkEnvironment
+  - EC2VPC
+  - EC2Instance
+  - EC2Volume
+  - EC2Subnet
+  - EC2RouteTable
+  - EC2NATGateway
+  - EC2InternetGateway
+  - EC2InternetGatewayAttachment
+  - EC2NetworkACL
+  - EC2SecurityGroup
+  - EC2NetworkInterface
+  - ECSService
+  - ECSCluster
+  - ELBLoadBalancer
+  - ELBv2
+  - ELBv2TargetGroup
+
+  # don't nuke IAM users
+  excludes:
+  - IAMUser
+
+accounts:
+# testing account
+  126450723953:
+    presets:
+    - defaults
+    - cpco
+
+presets:
+  defaults:
+    filters:
+      ECSCluster:
+      - type: "regex"
+        value: ".*cluster/fargate"
+      ECSService:
+      - type: "regex"
+        value: ".*service/atlantis"
+      EC2InternetGateway:
+      - property: "tag:Name"
+        type: "regex"
+        value: "^$"
+      EC2InternetGatewayAttachment:
+      - property: "tag:igw:Name"
+        type: "regex"
+        value: "^$"
+      EC2RouteTable:
+      - property: "tag:Name"
+        type: "regex"
+        value: "^$"
+      EC2Subnet:
+      - property: "DefaultForAz"
+        value: "true"
+      EC2VPC:
+      - property: "IsDefault"
```

Comment:
not sure how it works, but this (and the value: "true") will not nuke the default VPC?

Reply:
Ya, I'm ignore the default VPC. the `filters` section is everything to ignore.

Reply:
(and below `cpco-*` is always ignored)

Reply:
and everything else that's not the default or cpco (e.g. `eg-*` is nuked)